buildenv

```diff
@@ -31,7 +35,16 @@ zopen_append_to_env()
 
 zopen_append_to_setup()
 {
-  # echo commands that will run when installing via setup.sh
+cat <<EOF
```

Comment:
This might get lost in the noise of a larger install? This is the only program that requires root to install right? Not sure how to do it but at a minimum, maybe make sure the readme has in big letters that you need root to install sudo even if it's just into your own home directory

Reply:
Thanks, I have added a README and marked the requirement in bold